README.md

```diff
@@ -22,19 +22,14 @@ information using a variety of contributed modules and custom solutions.
 
 ## Install Instructions
 
-1. Clone the repository and install the project
+There are two ways to install Guardr.
```

Comment:
I think we can remove lines 25-27 next, talking about two ways to install Guardr. This will leave just the composer command under the install instructions.